### PR TITLE
(Studio) Update gradio and multicontrolnet UI.

### PR DESCRIPTION
**Pull request by monorimet**
Updates gradio pin to 4.7, which might just break everything apparently.

What else does this break? I'd like the new gradio features.

### CODE REVIEW

**Comment by monorimet** [comment]
FYI @one-lithe-rune you might have opinions about this change

**Comment by monorimet** [comment]
TODO: fix nod logo, see about controlnet issues.

File: apps/stable_diffusion/shark_sd.spec [hunk 1]
```diff
@@ -19,6 +19,9 @@ a = Analysis(
     win_private_assemblies=False,
     cipher=block_cipher,
     noarchive=False,
+    module_collection_mode={
+        'gradio': 'py',  # Collect gradio package as source .py files
```
Comment (dan-garvey) [inline apps/stable_diffusion/shark_sd.spec:23]: I dont even want to know how you figured out this was required...

File: apps/stable_diffusion/src/models/model_wrappers.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def process_vmfb_ir_sdxl(extended_model_name, model_name, device, precision):
     if "vulkan" in device:
         _device = args.iree_vulkan_target_triple
         _device = _device.replace("-", "_")
-        vmfb_path = Path(extended_model_name_for_vmfb + f"_{_device}.vmfb")
+        vmfb_path = Path(extended_model_name_for_vmfb + f"_vulkan.vmfb")
```
Comment (dan-garvey) [inline apps/stable_diffusion/src/models/model_wrappers.py:112]: intentional hardcode?

**Review by dan-garvey — APPROVED** [review]

File: apps/stable_diffusion/src/models/model_wrappers.py (hunk 2 above)
Comment (monorimet) [reply apps/stable_diffusion/src/models/model_wrappers.py:112]: for now. 
it should be target triple here but we need to change the naming convention globally

File: apps/stable_diffusion/shark_sd.spec (hunk 1 above)
Comment (monorimet) [reply apps/stable_diffusion/shark_sd.spec:23]: just wasnt collecting some random source files
tried adding metadata and hidden files, nope, ok just gimme it all then

**Pull request merged**